Laporan cek saldo nasabah

fahrizalrahman opened the pull request:
Nasabah dapat melakukan check saldo dan mutasi tabungan melalui form yang bisa diakses online di aplikasi minibank

fahrizalrahman committed:
Laporan cek saldo nasabah 

Nasabah dapat melakukan check saldo dan mutasi tabungan melalui form yang bisa diakses online di aplikasi minibank

RindangRamadhan reviewed (commented):
REVIEW
1. Indentasi dan penaman query lebih diperhatikan, agar memudahkan saat maintance
2. SELECT * diubah sesuai kebutuhan saja agar lebih ringan
	- cetak_lap_tabungan_rek.php
	- cetak_lap_tabungan_rek_nasabah.php
	- data_penyetoran.php
3. File db.php tidak perlu dicommit
4. kurangi query yang tidak diperlukan (jurusan.php)

RindangRamadhan committed:
Unplanned

- Pencegahan di Master Data > Nasabah, saat hapus jika nasabah tersebut sudah ada transaski maka tidak bisa dihapus.
- Perbaikan Form Hak Otoritas, Menampikan yang diperlukan saja.
- Menambahkan Otoritas Master Data > Jurusan, Transaksi Penyetoran Tabungan dan Transaksi Penarikan Tabungan.

RindangRamadhan committed:
Tambah Foto

- Foto di Index.php
- Foto di home.php

fahrizalrahman committed:
Menambahkan fiter reset passwrod dan penambahan master data akun kas dan posisi kas

fahrizalrahman committed:
Merge remote-tracking branch 'origin/test_minibank' into test_minibank

fahrizalrahman committed:
pengubahan nama penabungan -> penyetoran

RindangRamadhan committed:
Revisi Sprint Demo

- Memerpbaki kaporan tabungan nasabah

RindangRamadhan committed:
Merge remote-tracking branch 'origin/test_minibank' into test_minibank

RindangRamadhan committed:
Proses Login

Hapus echo

fahrizalrahman committed:
perbaikan laporan nasabah (tanggal default dan tampilan total)
